components/carousel/demo.js

```diff
@@ -55,3 +59,12 @@ document.querySelector('#change_page').addEventListener('keydown', (event) => {
     // enter
     if (event.keyCode === 13) document.querySelector('gameface-carousel').pageSize = Math.floor(event.currentTarget.value);
 });
+
+
+window.postMessage = function (message) {
+    pm({
+        origin: 'http://127.0.0.1/:3000',

```

Comment:
```suggestion
        origin: 'http://127.0.0.1:3000',
```
Should't be like that? :D

Reply:
This is the case for all of the components, I just made it to be consistent 😄